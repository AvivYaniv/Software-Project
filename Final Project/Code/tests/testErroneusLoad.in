edit tests/b2_3_fixedErroneusLoad.txt
solve tests/b2_3_fixedErroneusLoad.txt
validate
edit tests/b2_3_nonfixedErroneusLoad.txt
solve tests/b2_3_nonfixedErroneusLoad.txt
validate