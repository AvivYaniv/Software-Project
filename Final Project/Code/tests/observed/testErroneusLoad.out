Sudoku
------
Enter your command:
---------------------------
|  1   2*  3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3       5 |            |
---------------------------
|  5   2*    |            |
|  6         |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.      5.|            |
---------------------------
|  5.  2.    |            |
|  6.        |            |
---------------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
---------------------------
|  1   2*  3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3       5 |            |
---------------------------
|  5   2*    |            |
|  6         |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.      5.|            |
---------------------------
|  5.  2*    |            |
|  6.        |            |
---------------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
Exiting...
